homeassistant/components/pi_hole/strings.json

```diff
@@ -16,8 +16,16 @@
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
     },
     "abort": {
-      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]",
-      "duplicated_name": "Name already existed"
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "api_key": "API Key (Optional)"
```

Comment:
Please use the common key:
https://github.com/home-assistant/core/blob/af5374d38b3b3cf1950b0ac3959058db5a623ecd/homeassistant/strings.json#L36